frontend/src/views/AdvancedSearchPage/index.vue

```diff
@@ -82,6 +82,30 @@
                   class="!text-sm"
                 />
               </DsfrInputGroup>
+
+              <ElementAutocomplete
+                v-model="ingredientSearchTerm"
+                label="Composition"
+                label-visible
+                hint="Tapez au moins trois caractères pour démarrer la recherche. Les résultats contiendront tous les ingrédients ajoutés dans ce filtre."
+                @selected="addIngredient"
+                :hideSearchButton="true"
+                :required="false"
+                :chooseFirstAsDefault="false"
+              />
+              <div class="mt-2">
+                <DsfrTag
+                  v-for="([id, name, type], idx) in ingredientsToFilter"
+                  :key="`ingredient-${id}`"
+                  class="m-1 fr-tag--dismiss"
+                  @click="removeIngredient(idx)"
+                  :aria-label="`Retirer ${name}`"
+                  tagName="button"
+                >
+                  <v-icon scale="0.85" class="mr-2" :name="getTypeIcon(type)" :aria-label="getTypeInFrench(type)" />
```

Comment:
```suggestion
                  <v-icon scale="0.85" class="mr-1" :name="getTypeIcon(type)" :aria-label="getTypeInFrench(type)" />
```
mr-2
![Screenshot 2025-03-25 at 16-41-20 Recherche avancée - Compl'Alim](https://github.com/user-attachments/assets/2e8829b6-5379-4610-988c-6909bc2921fc)

mr-1
![Screenshot 2025-03-25 at 16-41-39 Recherche avancée - Compl'Alim](https://github.com/user-attachments/assets/0bc20b86-ebc7-4eed-9777-bb17abd77e59)